.gitlab-ci.yml

```diff
@@ -275,7 +279,8 @@ run-certification-tests:
     DEVICE_UNDER_TEST: prplmesh
   script:
       - |
-        if [ "$DEVICE_UNDER_TEST" != prplmesh ] ; then
+        if [ "$DEVICE_UNDER_TEST" = netgear-rax40 ] ; then
```

Comment:
Maybe it's better to just check that OPENWRT_IMAGE is set (and use OPENWRT_IMAGE below)?